evap/evaluation/models.py

```diff
@@ -470,7 +470,10 @@ def evaluation_ends_soon(self):
 
     @property
     def days_until_evaluation(self):
-        return (self.vote_start_datetime.date() - date.today()).days
+        days_left = (self.vote_start_datetime.date() - date.today()).days
+        if days_left == 0 and self.vote_start_datetime < datetime.now():
+            return -1
+        return days_left
```

Comment:
Would `days_left = (self.vote_start_datetime - datetime.now()).days` also work? I'd like that more.

Reply:
that would be a semantic change, since right now the `days_until_evaluation` tell you whether the evaluation starts today or tomorrow, and not whether it starts in up to 24 hours or up to 48 hours.

Reply:
Hmm... I think its unintuitive to have this conditional return value. I guess I'd have split this into two methods, one returns calendar days left, one returns full days left. If you both think it's okay we can leave it like this, though.

Reply:
used proposed solution from the other discussion about the method